pkg/provider/kubernetes/metadata.go

```diff
@@ -26,8 +26,9 @@ type PushSecretMetadataSpec struct {
 	TargetMergePolicy targetMergePolicy `json:"targetMergePolicy,omitempty"`
 	SourceMergePolicy sourceMergePolicy `json:"sourceMergePolicy,omitempty"`
 
-	Labels      map[string]string `json:"labels,omitempty"`
-	Annotations map[string]string `json:"annotations,omitempty"`
+	Labels          map[string]string `json:"labels,omitempty"`
+	Annotations     map[string]string `json:"annotations,omitempty"`
+	RemoteNamespace string            `json:"remoteNamespace,omitempty"`
```

Comment:
I know you didnt start with this. But I really think these spec fields deserve documentation. WDYT?

Reply:
You're right, I probably should update the doc and add the metadata settings. Thanks.

Reply:
Actually these are documentet. I just need to add RemoteNamespace to it.